### DISCUSSION TITLE
Zapier Integration returns empty transcription

### DISCUSSION THREAD

**Discussion by joep-this** [deepgram/community#856] [General help] [answered]
Here's the info from Deepgram below.  It was sent an mp3URL but didn't transcribe it.  I have this setup from a zapier integration using create transcription (text) action.  How should I go about debugging ?  

Using this from zapier: 

Model
nova-2
Language
da
Smart Format
true

From Deepgram usage:

Request Information
Method: 
POST
URL/Path: 
https://api.deepgram.com/v1/listen
Params: 
smart_format=true&model=nova-2&language=da
Duration (milliseconds): 
2801
Data: 
{
  "url": "https://api.twilio.com/2010-04-01/Accounts/ACdfa212cecdfd468ee436745205d05c4b/Recordings/RE57bd6ff02e7914d7f2c898c3dda603b9"
}
Response Information
Status code: 
200
URL/Path: 
https://api.deepgram.com/v1/listen
Content: 
{
  "metadata": {
    "transaction_key": "deprecated",
    "request_id": "6ed70162-298c-433c-9002-17617a1eac5f",
    "sha256": "03be83d47c3c5e8c435b2ba625985482ebb17a04a37ae91f9107e7a28ed0cab8",
    "created": "2024-07-18T10:52:59.637Z",
    "duration": 28.251938,
    "channels": 1,
    "models": [
      "71a7b6c7-4025-4cc9-949e-7524698d1fda"
    ],
    "model_info": {
      "71a7b6c7-4025-4cc9-949e-7524698d1fda": {
        "name": "2-general-nova",
        "version": "2024-01-06.6304",
        "arch": "nova-2"
      }
    }
  },
  "results": {
    "channels": [
      {
        "alternatives": [
          {
            "transcript": "",
            "confidence": 0,
            "words": [],
            "paragraphs": {
              "transcript": "\n",
              "paragraphs": []
            }
          }
        ]
      }
    ]
  }
}

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
Hi @joep-this 

I don't think the problem is with Zapier. I tried transcribing this audio with the following CURL request, and it did not work. It does work when transcribing it in English (since the English recording part of it is very clear), but the Danish audio quality combined with the English audio on top of it seems to have caused a problem for the model. The first part of the audio can have a strong effect on how the model understands the language of the audio, and since the English is very clear at the start, it only is able to transcribe the English. Have you had luck with other similar audio recordings?

```
curl \
  --request POST \
  --header 'Authorization: Token YOUR_API_KEY' \
  --header 'Content-Type: application/json' \
  --data '{"url":"https://api.twilio.com/2010-04-01/Accounts/ACdfa212cecdfd468ee436745205d05c4b/Recordings/RE57bd6ff02e7914d7f2c898c3dda603b9"}' \
  --url 'https://api.deepgram.com/v1/listen?model=nova-2&smart_format=true&language=da'
```
